Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -107,6 +107,19 @@ namespace gdjs {
       return true;
     }
 
+    onDestroy() {
```

Comment:
@arthuro555 FYI, when deleting an object with a tween behavior, we were not cleaning the tweens running in Shifty on this behavior. It was not easy to spot before the refactor because we were calling removeTween on a tween that was just finishing (so we were not leaking this tween, but potentially leaking *other* tweens going on the object!).

`onDestroy` is guaranteed to be called when the object is removed, and so it's the ideal place to clean all the tweens. So we both fix the issue of the tween wrongly continuing to apply after it's supposed to delete the object, and also fixing the other tweens that could also run on it.